### DISCUSSION TITLE
What happened to the nice --iso-8601 naming pattern of images taken with Markor?

### DISCUSSION THREAD

**Discussion by ghost** [gsantner/markor#2069] [General] [answered]
I am taking all my photos with Markor, and I just found that at some point the wonderful

```
Markor_$(date --iso=seconds).jpm
```
pattern has changed into an obscure 

```
IMG_YYYYMMDD_HHMMSS.jpg
```
pattern. Is there a reason for that?

**Comment by gsantner** [comment, marked as answer]
@lockywolf  
yes. So the sorting works by default together with any other image you shoot with the camera. Pretty much every camera software saves in the filename format with `IMG_` 😉 .
